<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookSettings</class>
 <widget class="QWidget" name="BookSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Table</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="GuiUtils::FontPushButton" name="btn_font">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Font Color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="GuiUtils::ColorPushButton" name="btn_fontColor">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Data Area</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="8" column="1">
           <widget class="QCheckBox" name="cb_alternateColor">
            <property name="text">
             <string>Alternate Color</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Data Area Background</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="GuiUtils::ColorPushButton" name="btn_DatabackgroundColor">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Display</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Row Number</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cbo_display">
            <item>
             <property name="text">
              <string>Decimal:1000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Scientific:1E3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Decimal:1,000</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="sbox_Number">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Column Width</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="sbx_columnWidth">
            <property name="minimum">
             <number>30</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="sbx_decimalPlace"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Decimal Places</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Table Header</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QCheckBox" name="cb_Unit">
            <property name="text">
             <string>Unit</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="cb_Comment">
            <property name="text">
             <string>Comment</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="cb_User">
            <property name="text">
             <string>User Function</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="cb_Filter">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="cb_LongName">
            <property name="text">
             <string>Long Name</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="cb_Sparklines">
            <property name="text">
             <string>Thumbnails</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="HeaderColor">
            <property name="text">
             <string>Table Header Background</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="GuiUtils::ColorPushButton" name="btn_HeaderColor">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GuiUtils::ColorPushButton</class>
   <extends>QPushButton</extends>
   <header>comguiutils/colorpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>GuiUtils::FontPushButton</class>
   <extends>QPushButton</extends>
   <header>comguiutils/fontpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>checkBoxClicked(bool)</slot>
 </slots>
</ui>
